### ISSUE TITLE
Postrges test fails consistently

### ISSUE THREAD

**Issue by Rots** [thrasher-corp/gocryptotrader#567]
# New Issue

## Context

* Operating System: Linux Mint 20

* GoCryptoTrader version (`gocryptotrader -version`): 
GoCryptoTrader v0.1 amd64 go1.15.1 pre-release.

## Expected Behavior

Running go tests locally pass.
If the target environment is not available for some tests, they should be skipped.

## Current Behavior

tests consistently fail

## Failure Information (for bugs)
`make test` fails

### Steps to Reproduce / Failure Logs

```
gocryptotrader$ go test -v ./database/models/postgres/
Unable to execute setup: parameter validation failed:
        Parameter is an empty string: psql.user
        Parameter is an empty string: psql.host
        Parameter is an empty string: psql.dbname
FAIL    github.com/thrasher-corp/gocryptotrader/database/models/postgres        0.005s
FAIL
```

**Comment by xtda** [comment]
yeah this is caused by it looking for either sqlboiler.json or env vars on start up due to usage of viper

there are a few other minor issues with the template generation from [our fork of sqlboiler](https://github.com/thrasher-corp/sqlboiler) such as a few of the other tests it will generate with FK relations will always fail and style related

I have been meaning to go through and update the fork to keep it inline with master
I think in the end i ended up finding local workrounds for almost all the issues besides one migration related so shouldn't be that much of an issue to keep it inline and also apply a fix to skip tests a lot more will get on it at some point this week hopefully
